.travis.yml

```diff
@@ -3,12 +3,13 @@ cache:
   directories:
     - WcaOnRails/vendor/bundle
 rvm:
-  - 2.3.1
+  - 2.4.0
```

Comment:
Unfortunately, adding a `ruby "2.4.0"` directive to the Gemfile [didn't work for travis](https://travis-ci.org/thewca/worldcubeassociation.org/builds/209352464#L394-L397):

```
Warning: the running version of Bundler (1.13.6) is older than the version that created the lockfile (1.14.6). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
Your Ruby version is 2.2.5, but your Gemfile specified 2.4.0
The command "eval bundle install --jobs=3 --retry=3 --deployment " failed. Retrying, 2 of 3.
```

I'll try out a `.ruby-version` file now.